packages/block-library/src/file/index.js

```diff
@@ -208,6 +208,10 @@ export const settings = {
 						href={ textLinkHref }
 						target={ textLinkTarget }
 						rel={ textLinkTarget ? 'noreferrer noopener' : false }
+						// ensure download attribute is still set when fileName
+						// is undefined. Using '' here as `true` still leaves
+						// the attribute unset.
+						download={ fileName || '' }
```

Comment:
This make sense to me. Though changing the HTML output of any block means we need to provide a "deprecated version" to ensure old blocks saved with the old markup are "upgraded" when we open the post in the editor.

See https://wordpress.org/gutenberg/handbook/block-api/deprecated-blocks/

Also, make sure to check the unit tests, there's probably a fixture to update.

Reply:
I understand, however, I am web designer, and I'm not the knowledge to do this… I read the documentation but I don't know where to start.

Reply:
That's fine, I'll start to help when I have some time. The idea is that you'll have to copy the previous "save" function and "attributes" definition into a "deprecated version"

Reply:
Oh I noticed this change causes an error where the block become invalid after saving it and refreshing the page. Maybe there's a good reason for omitting the download attribute here (it's present in the download button). So I'm pinging @noisysocks for more info.